skcosmo/preprocessing/FlexibleScaler.py

```diff
@@ -4,30 +4,20 @@
 
 
 class StandardFlexibleScaler(TransformerMixin, BaseEstimator):
-    """A more flexible scaler for features and data, that mimics
-    sklearn.preprocessing.StandardScaler make the mean of the
-    columns equal to zero and the variance of each column
-    (if designated) equal to one.
+    """Standardize features by removing the mean and scaling to unit variance.
+    Make the mean of the columns equal to zero and the
+    variance of each column (if designated) equal to one.
 
-    :param with_mean: If True, center the data before scaling
+    :param with_mean: If True, center the data before scaling. If False, save the mean the same
     :type with_mean: boolean
-    :param with_std: If True, scale the data to unit variance
+    :param with_std: If True, scale the data to unit variance. If False, save the variance the same
     :type with_std: boolean
-    :param column_wise: If True, normalize each column separately
+    :param column_wise: If True, normalize each column separately. If False, normalize the whole matrix, divided it by variaton.
     :type column_wise: boolean
     """
 
-    def __init__(self, *, with_mean=True, with_std=True, column_wise=False):
-        """Initialize StandardFlexibleScaler.
-
-
-        :param with_mean: If True, center the data before scaling
-        :type with_mean: boolean
-        :param with_std: If True, scale the data to unit variance
-        :type with_std: boolean
-        :param column_wise: If True, normalize each column separately
-        :type column_wise: boolean
-        """
+    def __init__(self, with_mean=True, with_std=True, column_wise=False):
```

Comment:
Let's pass the tolerance (used later to define 0 variance columns/matrices) as a parameter that the user can designate.

```suggestion
    def __init__(self, with_mean=True, with_std=True, column_wise=False, tol=1E-15):
```